presto-spi/src/main/java/com/facebook/presto/spi/security/SystemAccessControl.java

```diff
@@ -285,18 +285,40 @@ default void checkCanSetCatalogSessionProperty(Identity identity, String catalog
      *
      * @throws com.facebook.presto.spi.security.AccessDeniedException if not allowed
      */
+    @Deprecated
```

Comment:
The new default methods will fail if no one implements them, so this isn't backwards compatible. Thus, since plugins will need code changes anyway, so we should delete the old methods rather than marking them deprecated.